src/components/meta.pug

```diff
@@ -0,0 +1,34 @@
+if article
+  dl.meta
+    dt Publisher/s
+    dd
+      span #[+link('#', 'Name of publisher')], #[+link('#', 'Name of second publisher')]
```

Comment:
Should each publisher be wrapped in individual span tags?

Reply:
Not sure why a span was included, will remove this.